src/main/java/duke/commands/Command.java

```diff
@@ -0,0 +1,25 @@
+package duke.commands;
+
+import duke.data.TaskList;
+import duke.storage.Storage;
+import duke.ui.Ui;
+
+/**
+ * Class representing an executable Command.
+ */
+public abstract class Command {
+
+    /**
+     * Executes the command.
+     * @param tasks Current list of to-do-list tasks.
```

Comment:
Same as above, should have an empty line between the description section and parameters section!

Reply:
I like that there was punctuation and naming of the parameters are apropriate, eg 'tasks' to represent the list of tasks